tests/foreman/api/test_registration.py

```diff
@@ -0,0 +1,71 @@
+"""Tests for registration.
+
+:Requirement: Registration
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Registration
+
+:CaseAutomation: Automated
+
+:CaseImportance: Critical
+
+:Assignee: sbible
+
+:TestType: Functional
+
+:Upstream: No
+"""
+import pytest
+
+pytestmark = pytest.mark.tier1
```

Comment:
@jyejare Do we still categorize tests into tiers? Did we not deprecate tiers? Although, there is no harm in keeping them.

Reply:
Yes, we are still supporting tiers, not sure if there's any plan to deprecate, but would prefer this topic for JFT, will resolve this thread